Dockerfile

```diff
@@ -3,8 +3,15 @@ FROM python:3.7.2
 # We copy the rest of the codebase into the image
 RUN mkdir /harvester
 WORKDIR /harvester
+
+# Flask environment is set to dev for debug purposes
+ENV FLASK_ENV development
+
+# update apt-get and install psql
 RUN apt-get update -qq
 RUN apt-get install -y postgresql-client
+
 COPY requirements.txt /harvester/requirements.txt
+
 # Install Python dependencies
-RUN pip install -r requirements.txt
+RUN pip3 install -r requirements.txt
```

Comment:
Just so everyone is aware `FROM python:3.7.2` will set `python` and `pip` to `python3` and `pip3`

Reply:
good to know.  I can change it back